docs/user-guide/walkthrough/README.md

```diff
@@ -12,77 +12,176 @@ certainly want the docs that go with that version.</h1>
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-# Kubernetes 101 - Walkthrough
+# Kubernetes 101 - Kubectl CLI & Pods
+
+For Kubernetes 101, we will cover kubectl, pods, volumes, and multiple containers
+
+In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/GoogleCloudPlatform/kubernetes/releases) or [the source](https://github.com/GoogleCloudPlatform/kubernetes).
+
+Table of Contents
+- [Kubectl CLI](#kubectl-cli)
+  - [Install Kubectl](#install-kubectl)
+  - [Configure Kubectl](#configure-kubectl)
+- [Pods](#pods)
+  - [Pod Definition](#pod-definition)
+  - [Pod Management](#pod-management)
+  - [Volumes](#volumes)
+  - [Multiple Containers](#multiple-containers)
+- [What's Next?](#whats-next)
+
+
+## Kubectl CLI
+
+The easiest way to interact with Kubernetes is via the command-line interface.
+
+If you downloaded a pre-compiled release, kubectl should be under `platforms/<os>/<arch>`.
+
+If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
+
+For more info about kubectl, including its usage, commands, and parameters, see the [kubectl CLI reference](../kubectl/kubectl.md).
+
+#### Install Kubectl
+
+The kubectl binary doesn't have to be installed to be executable, but the rest of the walkthrough will assume that it's in your PATH.
+
+The simplest way to install is to copy or move kubectl into a dir already in PATH (like `/usr/local/bin`).
+
+An alternate method, useful if you're building from source and want to rebuild without re-installing is to use `./cluster/kubectl.sh` instead of kubectl. That script will auto-detect the location of kubectl and proxy commands to it (ex: `./cluster/kubectl.sh cluster-info`).
+
+#### Configure Kubectl
+
+If you used `./cluster/kube-up.sh` to deploy your Kubernetes cluster, kubectl should already be locally configured.
+
+By default, kubectl configuration lives at `~/.kube/config`.
+
+If your cluster was deployed by other means (e.g. a [getting started guide](../../getting-started-guides/README.md)), you may want to configure the path to the Kubernetes apiserver in your shell environment:
+
+```sh
+export KUBERNETES_MASTER=http://<ip>:<port>/api
+```
```

Comment:
Can we simply point people to https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/sharing-clusters.md#manually-generating-kubeconfig?

it's not using env var though.